lib/import-export-visitor.js

```diff
@@ -347,11 +394,21 @@ module.exports = class ImportExportVisitor extends Visitor {
       !! getOption(options, "generateLetDeclarations");
     this.modifyAST = !! getOption(options, "ast");
     this.parse = getOption(options, "parse");
+    this.enforceStrictMode = getOption(options, "enforceStrictMode");
```

Comment:
Is it odd that `enforceStrictMode` is `true` by default while others are `false`. Should this be flipped to loose or sloppy (I honestly don't know which is technically more correct).